PlayerScripts/PlayerScriptsPlace/StarterPlayer/StarterPlayerScripts/CameraScript/RootCamera.rbxmx

```diff
@@ -260,6 +261,22 @@ local function CreateCamera()
 	function this:GetRenderCFrame(part)
 		GetRenderCFrame(part)
 	end
+	
+	local STATE_DEAD = Enum.HumanoidStateType.Dead
+	
+	-- HumanoidRootPart when alive, Head part when dead
+	local function getHumanoidPartToFollow(humanoid, humanoidStateType)
+		if humanoidStateType ~= STATE_DEAD then
```

Comment:
can we flip this? doing a negative comparison generally makes stuff harder to read, especially when it seems easy to just use ==